### DISCUSSION TITLE
Variants.. how to make custom

### DISCUSSION THREAD

**Discussion by tbillion** [earlephilhower/arduino-pico#1318] [Q&A]
this will be my first excursion into this , I have a couple of different boards that i have fabbed up and i was wondering what the steps were in order to add a variant of my own to my personal copy of the code. mainly i just want it to be able to remap what busses are on what pins and have it accessible via the Arduino IDE instead of having to manually change the file back and forth all the time ?

**Comment by earlephilhower** [comment]
https://github.com/earlephilhower/arduino-pico/blob/master/docs/contrib.rst#adding-a-new-board
